dns.go

```diff
@@ -89,6 +89,9 @@ type DNSListResponse struct {
 	ResultInfo `json:"result_info"`
 }
 
+// listDNSRecordsDefaultPageSize represents the default per_page size of the API.
+var listDNSRecordsDefaultPageSize int = 100
```

Comment:
```suggestion
const listDNSRecordsDefaultPageSize int = 100
```

Reply:
Ah, sorry, I did not realize you intentionally made a variable so that the testing can be done without generating >100 entries.